### DISCUSSION TITLE
How to achieve resource isolation and access control in multi tenant mode in MinIO?

### DISCUSSION THREAD

**Discussion by hubcomsix** [minio/minio#18698] [Q&A] [answered]
## ExpectedBehavior
Implement resource isolation in multi tenant mode, ensuring that each tenant can only access their own data.

Implement access control to ensure that each tenant can only perform specific operations.

Ensure the stability and security of the system, prevent unauthorized access and data leakage.

## CurrentBehavior
Using MinIO's multi tenant feature: MinIO itself supports multi tenant functionality and can enable multi tenant mode through configuration files or environment variables. In multi tenant mode, MinIO assigns independent namespaces to each tenant for storing and managing tenant data.

**Comment by klauspost** [comment, marked as answer]
Please reach out on `sales@min.io` and we can go through what options there are available.
